interface/forms/annotate_diagram/selectnew.php

```diff
@@ -0,0 +1,154 @@
+<?php 
+/**
+ * Copyright Medical Information Integration,LLC info@mi-squared.com
+ * 
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * Rewrite and modifications by sjpadgett@gmail.com Padgetts Consulting 2016.
+ *
+ * @package OpenEMR
+ * @author  Medical Information Integration,LLC <info@mi-squared.com>
+ * @author  Jerry Padgett <sjpadgett@gmail.com> 
+ * @author  Terry Hill <terry@lillysystems.com>
+ * @link    http://www.open-emr.org
+ */
+
+
+include_once("../../globals.php");
+require_once($GLOBALS['srcdir'].'/api.inc');
+	$sanitize_all_escapes=true;
+	$fake_register_globals=false;
+	$images_dir = '../../forms/annotate_diagram/diagram/';
+	$images_per_row = 7;
+?>
+<html>
+<head>
+<title><?php xl('Select New Diagram for Annotation','e'); ?></title>
```

Comment:
remove e parameter and use echo xlt:
``````
<title><?php echo xlt('Select New Diagram for Annotation'); ?></title>